cli/src/commit_templater.rs

```diff
@@ -1987,6 +2016,25 @@ fn builtin_repo_path_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo, R
     map
 }
 
+fn builtin_diff_path_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo, DiffPath> {
+    let mut map = CommitTemplateBuildMethodFnMap::<DiffPath>::new();
+    map.insert(
+        "display",
+        |language, _diagnostics, _build_ctx, self_property, function| {
+            function.expect_no_arguments()?;
+            let path_converter = language.path_converter;
+            let out_property = self_property.map(move |path| match path {
+                DiffPath::Plain(path) => path_converter.format_file_path(&path),
+                DiffPath::Copied(path) => {
+                    path_converter.format_copied_path(path.source(), &path.target)
+                }
+            });
+            Ok(out_property.into_dyn_wrapped())
+        },
+    );
```

Comment:
We'll probably need `.source()`/`.target() -> RepoPath` accessors to allow `diff_path.target().absolute()`, etc.?